sdk/typescript/test/unit/cryptography/multisig.test.ts

```diff
@@ -98,3 +103,179 @@ describe('multisig address and combine sigs', () => {
 		).toEqual(false);
 	});
 });
+
```

Comment:
these methods are deprecated, and will be removed pretty soon, but no harm in testing them until they are removed